Resources/Prototypes/Roles/Jobs/Security/brigmedic.yml

```diff
@@ -19,3 +22,20 @@
   special:
   - !type:AddImplantSpecial
     implants: [ MindShieldImplant, TrackingImplant ]
+
+- type: startingGear
+  id: BrigmedicGear
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitBrigmedic
+    outerClothing: ClothingOuterArmorPlateCarrier
```

Comment:
This doesn't seem right... This plate carrier doesn't seem to exist, sure you didn't confuse it with another armor vest?